src/main/java/uk/ac/ebi/ena/webin/cli/validator/manifest/Manifest.java

```diff
@@ -42,6 +42,8 @@ public abstract class Manifest <FileType extends Enum<FileType>> {
     private boolean quick;
     private boolean ignoreErrors;
     private String webinAuthToken;
+    private String webinRestUri;
+    private String biosamplesUri;
     private boolean webinCliTestMode;
```

Comment:
Please remove the webinCliTestMode. We are replacing this option with explicitly setting the URLs (webinRestUri, biosamplesUri).